docs/resources/release.md

```diff
@@ -0,0 +1,333 @@
+- `disable_crd_hooks` (Boolean) Prevent CRD hooks from, running, but run other hooks.  See helm install --no-crd-hook
+- `disable_openapi_validation` (Boolean) If set, the installation process will not validate rendered templates against the Kubernetes OpenAPI Schema
+- `disable_webhooks` (Boolean) Prevent hooks from running.
+- `force_update` (Boolean) Force resource update through delete/recreate if needed.
+- `keyring` (String) Location of public keys used for verification. Used only if `verify` is true
+- `lint` (Boolean) Run helm lint when planning
+- `max_history` (Number) Limit the maximum number of revisions saved per release. Use 0 for no limit
+- `namespace` (String) Namespace to install the release into.
+- `pass_credentials` (Boolean) Pass credentials to all domains
+- `postrender` (Block List, Max: 1) Postrender command configuration. (see [below for nested schema](#nestedblock--postrender))
+- `recreate_pods` (Boolean) Perform pods restart during upgrade/rollback
+- `render_subchart_notes` (Boolean) If set, render subchart notes along with the parent
+- `replace` (Boolean) Re-use the given name, even if that name is already used. This is unsafe in production
+- `repository` (String) Repository where to locate the requested chart. If is a URL the chart is installed without installing the repository.
+- `repository_ca_file` (String) The Repositories CA File
+- `repository_cert_file` (String) The repositories cert file
+- `repository_key_file` (String) The repositories cert key file
+- `repository_password` (String, Sensitive) Password for HTTP basic authentication
+- `repository_username` (String) Username for HTTP basic authentication
+- `reset_values` (Boolean) When upgrading, reset the values to the ones built into the chart
+- `reuse_values` (Boolean) When upgrading, reuse the last release's values and merge in any overrides. If 'reset_values' is specified, this is ignored
+- `set` (Block Set) Custom values to be merged with the values. (see [below for nested schema](#nestedblock--set))
+- `set_list` (Block List) Custom sensitive values to be merged with the values. (see [below for nested schema](#nestedblock--set_list))
+- `set_sensitive` (Block Set) Custom sensitive values to be merged with the values. (see [below for nested schema](#nestedblock--set_sensitive))
+- `skip_crds` (Boolean) If set, no CRDs will be installed. By default, CRDs are installed if not already present
+- `timeout` (Number) Time in seconds to wait for any individual kubernetes operation.
+- `values` (List of String) List of values in raw yaml format to pass to helm.
+- `verify` (Boolean) Verify the package before installing it.
+- `version` (String) Specify the exact chart version to install. If this is not specified, the latest version is installed.
+- `wait` (Boolean) Will wait until all resources are in a ready state before marking the release as successful.
```

Comment:
I see that this new Markdown doc no longer includes the default values for variables. The defaults were documented in the prior doc versions.